.github/workflows/build_test_publish.yml

```diff
@@ -7,6 +7,9 @@ on:
   push:
     branches:
       - main
+    paths:
```

Comment:
If we run this on push to branches other than `main` it will run on the fork with each push, and the author can see whether it works or not

Reply:
You're right, I'll remove the branch requirement so the workflow is triggered on pushes to any branch (assuming the `path` requirement is met)